<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>leftDock</class>
 <widget class="QDockWidget" name="leftDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>675</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>172</width>
    <height>415</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>2000</height>
   </size>
  </property>
  <property name="floating">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Side Menu</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lableName">
      <property name="text">
       <string>Desination Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inputName"/>
    </item>
    <item>
     <widget class="QLabel" name="labelState">
      <property name="text">
       <string>State</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inputState"/>
    </item>
    <item>
     <widget class="QLabel" name="labelDate">
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>Date Visited</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="inputDate"/>
    </item>
    <item>
     <widget class="QLabel" name="labelFun">
      <property name="text">
       <string>Fun Level</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>12</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDial" name="inputFun">
      <property name="maximumSize">
       <size>
        <width>450</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="pageStep">
       <number>0</number>
      </property>
      <property name="value">
       <number>27</number>
      </property>
      <property name="sliderPosition">
       <number>27</number>
      </property>
      <property name="wrapping">
       <bool>false</bool>
      </property>
      <property name="notchTarget">
       <double>10.000000000000000</double>
      </property>
      <property name="notchesVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="displayData">
      <property name="cursor" stdset="0">
       <cursorShape>ForbiddenCursor</cursorShape>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="submitBtn">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
